City-of-Helsinki/berth-reservations-admin
# VEN-852 | Creating invoices for unmarked winter storage notices

## Description :sparkles:

* Create invoices for unmarked winter storage notices 
* Extract generic parts from OfferCard to InvoiceCard, rename OfferCard to BerthOfferCard
* Add status to unmarked ws notice view

## Issues :bug:

### Closes :no_good_woman:

* [VEN-852](https://helsinkisolutionoffice.atlassian.net/browse/VEN-852): Creating invoices for unmarked winter storage notices

## Testing :alembic:

### Automated tests :gear:️

* Updated existing tests
* Added tests for new components

### Manual testing :construction_worker_man:

1. Create an unmarked winter storage notice from the customer UI (with your own email)
2. Locate the notice in the admin UI
3. Connect/create a customer to the notice (with your own email)
4. Create the invoice
5. Send the invoice
